README.md

```diff
@@ -192,134 +82,29 @@ local on_attach = function(client, bufnr)
   buf_set_keymap('n', '[d', '<cmd>lua vim.lsp.diagnostic.goto_prev()<CR>', opts)
```

Comment:
What these each do is pretty obvious to someone who has used the LSP client before, but inline comments here might be useful? Obviously all of this can be found in the _Wiki_ as well, but just to entice people a bit more to the client itself (understanding that this is indeed just wrapping config for it, but to most it provides their first experience with the client at all).

Reply:
Would a see `:help vim.lsp.*` suffice? The built-in docs are quite good

Reply:
I'd absolutely add that, but I think part of what you've mentioned around people thinking that the builtin LSP client is "hard to use" comes from a lack of understand to how much is already there for you. So here, I think adding comments like "move back and forth between server-reported diagnostics" might help flip that switch by putting things in plain English?

Obviously this isn't necessary by any means, and duplicating what is already in `:h lsp` is useless, but I think it might help bridge the gap in understanding that some new users have.

Reply:
I'll debate what to add notes to, I added a small note to omnifunc and the thing about ':help`